settings.js

```diff
@@ -91,57 +91,63 @@ var settings = {
 			// Need to flash Bootloader and OTA Flag before system parts for Photon/P1
 			// because as soon as the system parts are flashed they will change the location
 			// of DFU read/write functions which live in system firmware.
-			otaRegion: 'photon-bootloader@1.3.1.bin',
+			otaRegion: 'photon-bootloader@1.4.0.bin',
 			otaFlag: 'ota-flag-a5.bin',
-			systemFirmwareOne: 'photon-system-part1@1.3.1.bin',
-			systemFirmwareTwo: 'photon-system-part2@1.3.1.bin'
+			systemFirmwareOne: 'photon-system-part1@1.4.0.bin',
+			systemFirmwareTwo: 'photon-system-part2@1.4.0.bin'
 		},
 		'2b04:d008': {
 			// Need to flash Bootloader and OTA Flag before system parts for Photon/P1
 			// because as soon as the system parts are flashed they will change the location
 			// of DFU read/write functions which live in system firmware.
-			otaRegion: 'p1-bootloader@1.3.1.bin',
+			otaRegion: 'p1-bootloader@1.4.0.bin',
 			otaFlag: 'ota-flag-a5.bin',
-			systemFirmwareOne: 'p1-system-part1@1.3.1.bin',
-			systemFirmwareTwo: 'p1-system-part2@1.3.1.bin'
+			systemFirmwareOne: 'p1-system-part1@1.4.0.bin',
+			systemFirmwareTwo: 'p1-system-part2@1.4.0.bin'
 		},
 		'2b04:d00a': {
 			// The bin files MUST be in this order to be flashed to the correct memory locations
-			systemFirmwareOne:   'electron-system-part2@1.3.1+debug.bin',
-			systemFirmwareTwo:   'electron-system-part3@1.3.1+debug.bin',
-			systemFirmwareThree: 'electron-system-part1@1.3.1+debug.bin'
+			systemFirmwareOne:   'electron-system-part2@1.4.0+debug.bin',
+			systemFirmwareTwo:   'electron-system-part3@1.4.0+debug.bin',
+			systemFirmwareThree: 'electron-system-part1@1.4.0+debug.bin'
```

Comment:
do you know why we appear to be using a `debug` bin for the `electron` but nowhere else?

Reply:
This is normal. By default during the release Electron binaries are built with `DEBUG_BUILD=y` to enable additional modem logs.

Reply:
is it for troubleshooting? why not do it for `boron` as well?

Reply:
Yes. For troubleshooting. Boron doesn't have enough flash space currently to build with `DEBUG_BUILD=y` and all the logs.